ext/strings.go

```diff
@@ -356,6 +377,11 @@ func (stringLib) ProgramOptions() []cel.ProgramOption {
 				Operator: "string_upper_ascii",
 				Unary:    callInStrOutStr(upperASCII),
 			},
+			&functions.Overload{
```

Comment:
You'll also want to create entries for the overload ids, similar to what's seen for `substring` relative to `string_substring_int` and `string_substring_int_int`

Reply:
Good catch. I wasn't sure what the naming convention should be for a list. I went with: `list_join` for `<list<string>>.join()` and `list_join_string` for `<list<string>>.join(<string>)`.

Reply:
Usually I use the top-level types, so you picked what I would have here.